From a7d9d2c5550ff81f8adb0a5269c8e906ad107ab4 Mon Sep 17 00:00:00 2001
From: sjayanti <satya.charitardha.jayanti@intel.com>
Date: Fri, 20 Jul 2012 14:39:41 +0530
Subject: [PORT FROM MAIN] Audio-MW: Last word getting truncated in GPS direction application

BZ: 22016

Original BZ: 10392

MediaPlayerService uses buffer count of 4, this workaround is to give support for
opencore engine added by google. Since we use Stagefright, this is not required.
For betterment of latency changing the buffer count to 2.

Change-Id: Ib7799715cf2076acfaab73d618adacd8f45010c3
Orig-Change-Id: I4aa1e8aa0034cc58f44683a3e8ebf2f5ccdc1766
Signed-off-by: sjayanti <satya.charitardha.jayanti@intel.com>
Reviewed-on: http://android.intel.com:8080/77895
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: M, Arulselvan <arulselvan.m@intel.com>
Tested-by: M, Arulselvan <arulselvan.m@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 media/libmediaplayerservice/MediaPlayerService.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/media/libmediaplayerservice/MediaPlayerService.cpp b/media/libmediaplayerservice/MediaPlayerService.cpp
index ec6ace1..7ce6d1f 100644
--- a/media/libmediaplayerservice/MediaPlayerService.cpp
+++ b/media/libmediaplayerservice/MediaPlayerService.cpp
@@ -195,7 +195,7 @@ static bool checkPermission(const char* permissionString) {
 }
 
 // TODO: Find real cause of Audio/Video delay in PV framework and remove this workaround
-/* static */ int MediaPlayerService::AudioOutput::mMinBufferCount = 4;
+/* static */ int MediaPlayerService::AudioOutput::mMinBufferCount = 2;
 /* static */ bool MediaPlayerService::AudioOutput::mIsOnEmulator = false;
 
 void MediaPlayerService::instantiate() {
-- 
1.7.4.1

